app/models/concerns/common_metadata.rb

```diff
@@ -15,6 +15,11 @@ module CommonMetadata
     property :source_metadata,
              predicate: ::RDF::URI.new('http://library.princeton.edu/terms/source_metadata'),
              multiple: false
+    property :state,
+             predicate: ::RDF::URI.new('http://fedora.info/definitions/1/0/access/objState'),
+             multiple: false do |index|
```

Comment:
Yeah, the previous property definitions are indented so their arguments are in a block (rubocop seems to recognize and allow this), but then the block must be indented two spaces.

Related, maybe we should define some RDF vocabs for these predicates, and then the property definitions might be shorter enough to fit on one line?